man/passwd.1.xml

```diff
@@ -65,60 +65,60 @@
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
     <para>
-      The <command>passwd</command> command changes passwords for user accounts.
-      A normal user may only change the password for their own account, while
-      the superuser may change the password for any account.
-      <command>passwd</command> also changes the account or associated
-      password validity period.
+      The <command>passwd</command> command changes passwords
+      for user accounts.
+      A regular user can only change the password for their own account,
+      while the superuser can change the password for any account.
+      The <command>passwd</command> also changes the account
+      or associated password validity period.
     </para>
 
     <refsect2 id='password_changes'>
       <title>Password Changes</title>
       <para>
-	The user is first prompted for their old password, if one is
-	present. This password is then encrypted and compared against the
-	stored password. The user has only one chance to enter the correct
-	password. The superuser is permitted to bypass this step so that
-	forgotten passwords may be changed.
+      The user is first prompted for their old password, if one is present.
+      This password is then encrypted and compared with the stored password.
+      The user has only one attempt to enter the correct password.
+      The superuser can bypass this step to allow changing forgotten passwords.
```

Comment:
Indentation error

Reply:
No, not really. I'd recommend you to use the same indentation as the next paragraph